pull_request_template.md

```diff
@@ -0,0 +1,7 @@
+## Summary
+
+[Change!] Describe your feature, problems you had, notes, improvements and others.
+
+## Screenshots
+
+[Change!] Show the screenshots of your changes.
```

Comment:
I don't understand this PR template. Screenshot of what?

Reply:
I agree with removing the Screenshots part

Reply:
:+1: